disnake/types/message.py

```diff
@@ -77,6 +77,19 @@ class RoleSubscriptionData(TypedDict):
     is_renewal: bool
 
 
+class GuildProductPurchase(TypedDict):
+    listing_id: Snowflake
+    product_name: str
+
+
+PurchaseType = Literal[0]
+
+
+class PurchaseNotification(TypedDict):
+    type: PurchaseType
+    guild_product_purchase: NotRequired[GuildProductPurchase]
+
+
 # fmt: off
 MessageType = Literal[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 15, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 36, 37, 38, 39]
```

Comment:
Can you add the new message type `44` here too?

Reply:
d279ee1